### DISCUSSION TITLE
Release 5.0.0 CoKreat Execution status

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-coKreat/Community#18] [General Discussions]
Hi All,

Please find the testing status of R5.0.0 as of 30th Aug 2022,

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress  (25.90% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)

**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Execution Status:**

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

% Pending Cokreat | % Executed Cokreat
-- | --
74.1 | 25.9

</body>
</html>

**Defect Status:**

![image](https://user-images.githubusercontent.com/40622169/187472516-1fa397cd-798f-4c44-b60c-9e8581b9f42a.png)

![image](https://user-images.githubusercontent.com/40622169/187472677-4a8ed242-5a07-4e5a-b151-06f68dbc7b70.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 01st Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (30.50% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Execution Status:

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

% Pending Cokreat | % Executed Cokreat
-- | --
69.5 | 30.5


</body>
</html>

Defect Status:

![image](https://user-images.githubusercontent.com/40622169/187937193-59f7ec0b-aa75-436f-a6fb-593ceda0699a.png)

![image](https://user-images.githubusercontent.com/40622169/187937281-27a188be-ad62-4551-a600-3d9f54d62805.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha

**Reply by ShailandraOjha** [reply]
Hi All,

Please find the testing status of R5.0.0 as of 02nd Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (35.58% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Execution Status:**
![image](https://user-images.githubusercontent.com/44188853/188196805-62deda0e-2e77-47ea-8d2b-fb1e35fe36ac.png)

**Defect Status:**
![image](https://user-images.githubusercontent.com/44188853/188194990-0a32f2ab-ee29-467b-b058-133291b45cf6.png)
![image](https://user-images.githubusercontent.com/44188853/188195102-f8114b99-23e4-41f2-bf65-15bd6679e869.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @anunagaraj @swayangjit

Regards,
Shailandra Ojha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of  05th Sep 2022,

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (39.20% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/188463317-f249a77b-6b38-4358-91fb-6c13a16219bf.png)

**Defect Status:**
![image](https://user-images.githubusercontent.com/40622169/188463536-dce0ad7e-a093-4337-9bec-23466fa10e95.png)

![image](https://user-images.githubusercontent.com/40622169/188463587-442a8bdd-d71a-4c4a-beb2-c669e8d58fda.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 06th Sep 2022,

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (40% Completed)
**Planned Sign-Off Date:** Sep 13, 2022
**Defects Dashboard:** [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/188659903-ae8cab51-f566-46f4-98ed-2e1506c8d845.png)

**Defect Status:**
![image](https://user-images.githubusercontent.com/40622169/188660055-c5055f09-2230-4309-aa02-b3c89e9b6748.png)

![image](https://user-images.githubusercontent.com/40622169/188660137-176c206f-b04c-4bf1-83f8-7981faf3c2a1.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 07th Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (40% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Execution Status:

![image](https://user-images.githubusercontent.com/40622169/188940792-0252f807-d476-4a96-9ad7-5efe5d90abf3.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/188940870-9539a0ac-644e-49c3-bfcc-12e5cdcc4d57.png)

![image](https://user-images.githubusercontent.com/40622169/188940921-8f9c0bf3-0489-4a6b-90d1-439230d77f50.png)


@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 08th Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (46.30% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/189177756-3bf32038-25b2-40b0-9a23-dc1d838de94c.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/189177817-b1097e88-235a-4307-8ce3-e431769eb81e.png)

![image](https://user-images.githubusercontent.com/40622169/189177874-2be0bf1f-c88a-404c-a922-2ea242a23e2b.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 09th Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (64.2% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/189379699-e2a1fd19-7519-45a1-b9db-0e5f4e02f558.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/189379841-947d0d27-b651-4129-8989-4dfceea12c74.png)

![image](https://user-images.githubusercontent.com/40622169/189379905-2982e915-f86e-4791-bea0-91a69130f86f.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 12th Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (78.4% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/189703390-00a7262c-3ac5-4536-8378-23154ac06870.png)

Open  Defects:
![image](https://user-images.githubusercontent.com/40622169/189703613-19532c77-a4ff-4ea2-b95e-9beb633402e4.png)

![image](https://user-images.githubusercontent.com/40622169/189703713-abecd08b-3c09-4446-997e-b533196d107c.png)


@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @ShailandraOjha @swayangjit

Regards,
Anusha

**Comment by ShailandraOjha** [comment]
Hi All,

Please find the testing status of R5.0.0 as of 12th Sep 2022,

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (91.3% Completed)
Planned Sign-Off Date: Sep 13, 2022
Defects Dashboard: [Link](https://project-sunbird.atlassian.net/jira/dashboards/10606)
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1ICOHcOtj1xSVBIRf3UtE3AHg_v6yNP11TZj1myGsJAs/edit#gid=1981725419)

**Note:**  We have 4 open issues, blocking 82 test cases.

**Execution Status:**
![image](https://user-images.githubusercontent.com/44188853/189939250-f617cfad-6c08-4f5f-b731-741af12c17b7.png)

**Defects Status:**
![image](https://user-images.githubusercontent.com/44188853/189939723-f8bc0b2f-31ef-4360-8ece-cb0b81f571f8.png)

@rajeevsathish @surendrasinghs @sharathkashyap @rhwarrier @anisha147 @luckynvdu @vinukumar-vs(https://github.com/vinukumar-vs) [@pallakartheekreddy](https://github.com/pallakartheekreddy) @reshmi-nair @anandp504 @gandham-santhosh@kameshbhr @mmanimegalai@madhucr @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @mmanimegalai @chitranshu-keshav [AdarshP29](https://github.com/AdarshP29)@Ak0094 @anandvarada @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @swayangjit 

Regards,
Shailandra Ojha
